cron_job.py

```diff
@@ -0,0 +1,18 @@
+from crontab import CronTab
+
+cron = CronTab()
+
+job = cron.new(command='python image_retrieval.py')
+job.minute.every(1)  # can change according to requirement (hours, week, month, etc.)
+
+cron.write()
+print("Job created")
+
+for job in cron:
+    print(job)
+
+cron.remove(job)
+print("Job removed")
+
+for job in cron:
+    print(job)
```

Comment:
Why printing job, remove job, and printing job again?

Reply:
Apologies. The print and remove statements were for debugging purposes. I have removed those now. I have also changed the cron job from per minute to per month as required.